beetsplug/musicbrainz.py

```diff
@@ -112,12 +92,42 @@ def get_message(self):
     "recording-rels",
     "release-rels",
 ]
-if "work-level-rels" in musicbrainzngs.VALID_BROWSE_INCLUDES["recording"]:
-    BROWSE_INCLUDES.append("work-level-rels")
 BROWSE_CHUNKSIZE = 100
 BROWSE_MAXTRACKS = 500
 
 
+class LimiterTimeoutSession(LimiterMixin, TimeoutSession):
+    pass
+
+
+@dataclass
+class MusicBrainzAPI(RequestHandler):
```

Comment:
Very nicely done! I very much enjoy this abstraction layer for defining all possible api requests.